packages/paste-core/components/typography/src/Paragraph.tsx

```diff
@@ -1,26 +1,19 @@
 import * as React from 'react';
+import * as PropTypes from 'prop-types';
 import {Text} from '@twilio-paste/text';
 
-interface Paragraph {
+export interface ParagraphProps {
```

Comment:
This should have been fine the old way, no?

Reply:
Reverted that change.